docs/content/Portal.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Portal
+---
+
+
+Portals allow you to create a separation between the logical React component hierarchy and the physical DOM. See the [React documentation on portals](https://reactjs.org/docs/portals.html) for an in-depth explanation.
+
+This Portal component will render all children into the portal root DOM node instead of as children of this Portal's parent DOM element. This is useful for breaking out of the current stacking context. For example, popup menus and tool tips may need to render on top of (read: covering up) other UI. The best way to guarantee this is to add these elements to top-level DOM, such as directly on document.body. These elements can then be moved to the correct location using absolute positioning.
+
+## Customizing the portal root
+By default, Primer will create a portal root for you as a child of `document.body`. If you would like to specify your own portal root, there are two options:
```

Comment:
Should this be updated to say that it's added by default to the `BaseStyles` component which all applications should have as high up as possible in their component tree?